packages/embark-utils/src/index.js

```diff
@@ -218,14 +220,17 @@ function buildUrlFromConfig(configObj) {
   return buildUrl(configObj.protocol, canonicalHost(configObj.host), configObj.port, configObj.type);
 }
 
+function joinPath() {
+  return path.join.apply(path.join, arguments);
+}
+
+function tmpDir(...args) { return joinPath(os.tmpdir(), ...args); }
 
 const Utils = {
   buildUrl,
   buildUrlFromConfig,
-  joinPath: function() {
-    const path = require('path');
-    return path.join.apply(path.join, arguments);
-  },
+  joinPath,
+  tmpDir,
```

Comment:
Still need to go through the codebase and check for `tmpDir` and `joinPath` usages to replace with imports from `embark-utils`